### DISCUSSION TITLE
Getting issue on AzureDevOpsBuildArtifact - Unexpected value for artifact type 'PipelineArtifact'

### DISCUSSION THREAD

**Discussion by santhoshmohanr** [cake-build/cake#3651] [Extension Q&A]
Hi Team,
I am trying to get AzurePipelineArtifactDetails using AzureDevOpsBuildArtifact class. I am getting following issue,
`An error occurred when executing task 'UnZipArtifact'.
Error: One or more errors occurred.
        Unexpected value for artifact type 'PipelineArtifact'`

Implementation : 

```
var buildsSettings = new AzureDevOpsBuildsSettings(new Uri("https://dev.azure.com/test/"),
         "Azure-Pipeline-Example", AzureDevOpsAuthenticationPersonalAccessToken("accesstoken"));
var builds = AzureDevOpsBuilds(buildsSettings);
foreach (var build in builds)
{
   Information("Build Id = {0} ", build.BuildId);
   var artifacts = build.GetArtifacts();
   if(artifacts != null){
        Information("Artifact");
        foreach (var item in artifacts)
        {
           Information("Artifact Id = {0} ", item.Id);
           Information("Artifact Name = {0} ", item.Name);
        }
         
    }else{
         Information("Empty Artifact");
    }
}
```

Please help here to resolve the issue.

**Comment by pascalberger** [comment]
Are you using pipeline artifacts in your builds? Seems like they are not known to the corresponding enum in Cake: https://cakebuild.net/api/Cake.Common.Build.AzurePipelines.Data/AzurePipelinesArtifactType/

**Reply by santhoshmohanr** [reply]
Thanks so much for your explanation.
Yes exactly. I need to download Azure Universal Packages artifact and need to use in builds. 
Earlier I used latest version of AzureDevOps. But later I reduced the version it's working fine. 

`Seems like they are not known to the corresponding enum in Cake: https://cakebuild.net/api/Cake.Common.Build.AzurePipelines.Data/AzurePipelinesArtifactType/`
- I just followed the example from here (https://cakebuild.net/api/Cake.AzureDevOps/AzureDevOpsAliases/238B1087). I am not sure about how to use AzurePipelinesArtifactType here. 
- Please give some explanation how to use here.
